components/Carousel.js

```diff
@@ -25,21 +26,24 @@ export default function Carousel({ slides }) {
         <CarouselProvider
           className={`${styles.carousel}`}
           infinite
-          isPlaying
+          loop
```

Comment:
I don't find `loop` in the documentation for `CarouselProvider`. Is this making any difference?